src/arctic3d/modules/sequence.py

```diff
@@ -1,24 +1,39 @@
+import logging
 import tempfile
 
+from Bio import Align
+from Bio.Align import substitution_matrices
 from pdbtools.pdb_tofasta import pdb_to_fasta
 
+import os
+import shutil
 
-def load_seq(fasta_file):
-    """
-    Load sequence.
-
-    Parameters
-    ----------
-    fasta_file : str
-        Fasta file.
-
-    Returns
-    -------
-    fasta_seq : str
-        Fasta sequence.
+log = logging.getLogger("arctic3d.log")
 
-    """
-    pass
+LETTERS = [
+    "A",
+    "B",
+    "C",
+    "D",
+    "E",
+    "F",
+    "G",
+    "H",
+    "I",
+    "K",
+    "L",
+    "M",
+    "N",
+    "P",
+    "Q",
+    "R",
+    "S",
+    "T",
+    "V",
+    "W",
+    "Y",
+    "Z",
+]
```

Comment:
You are missing some letters here, is it the intention? Also I'd do it in a more Python way:

```
import string
LETTERS = [letter for letter in string.ascii_uppercase if letter not in "JOUX"]
```

Reply:
good suggestion! no intention in it, just my copilot being weird